### PR TITLE
Azure runner GitHub fixes

### PR DESCRIPTION
**Pull request by jmdegeorge**
## Description

Updated system-summary-report and classpath adjustments to resolve errors when running Azure tests through github actions. 
Added README and cucumber feature cleanup. 
Added Gradle task for removing temporary project folders. 

## Motivation and Context

Adds support for test coverage reporting to SonarQube. Contributes to #143 

## Types of changes

| Yes/No |  Pull Request Type          |  Description                                                          |
| ------ | --------------------------- | --------------------------------------------------------------------- |
|        | Bug fix                     | non-breaking change which fixes an issue                              |
|        | New feature                 | non-breaking change which adds functionality                          |
|        | Breaking change             | fix or feature that would cause existing functionality to change      | 
|        | Documentation change        | non-breaking change which modifies or updates documentation           |
|        | New tests                   | new unit tests, test scenarios, or test case documentation            | 
|        | Triggers regression testing | change affects downstream modules and will require regression testing | 
|     Y   | Other                       |           Azure cucumber test fixes                                                            | 

## How Has This Been Tested?

Tested locally. Run test computes through GitHub actions. 

## Notes for Reviewers
Please consider the following when reviewing this PR:
- **Correctness:** Does the code do what it claims? Are edge cases handled appropriately?
- **Clarity:** Is the code readable, maintainable and aligned with **SOLID** design principles?
- **Impact:** Will this change affect other parts of the system? Any potential regressions?
- **Testing:** Are the test cases sufficient and appropriate? Are there gaps in coverage?
- **Documentation:** Does this require updates to code comments, README, or other docs?

### CODE REVIEW

File: wrims-comparison-test/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+### Location
+- Module path: `wrims-engine/wrims-comparison-test`
+
+---
+
+### Project layout overview
+- `build.gradle`
+    - Declares tasks and dependencies for tests, native libs, and report generation.
+    - Loads optional environment variables from a local `.env.cucumber` file.
```
Comment (zack-rma) [inline wrims-comparison-test/README.md]: Consider specifying the intended location of the environment file to be clear on proper setup.

File: wrims-comparison-test/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    - `azure/`
+        - Cucumber features that download test projects from Azure Storage and run computes, e.g.:
+            - `compare_calLite_41_TF.feature`
+            - `compute_calsim3-dcr-9.6.0-10y.feature`
+            - `compute_calsim3-dcr-9.6.0-short.feature`
+    - `comparison/`
+        - Features that compare outputs to reference results, e.g.:
+            - `compare_calLite_41_TF_results.feature`
+    - `comparisonInputFiles/`
+        - Inputs for the comparison report tool, e.g. `callite_version_check_dss6_6.inp`.
```
[the remark this answers is not in the document]
Comment (jmdegeorge) [reply wrims-comparison-test/README.md]: Fixed and added some more clarifications to the README

**Review by zack-rma — APPROVED** [review]
